CONTRIBUTING.md

```diff
@@ -2,156 +2,261 @@
 
 ## Running and Developing DB
 
+### Introduction 
+
+Welcome to the Platformatic development guide. This document will help you set up your development environment, run the [Platformatic DB](https://docs.platformatic.dev/docs/db/overview), and contribute to the project. Follow the steps below to get started.
+
 ### Preparation
 
-1. Clone this repository, and enter in the directory (`cd platformatic` on most installations)
-2. Install pnpm `npm i pnpm --location=global` (if you're on Mac, you can use `brew` to install it with `brew install pnpm`)
-3. Make sure to run `pnpm setup` to configure the local path; then close and reopen your terminal.
-4. Install dependencies `pnpm i`
-5. Install docker with Docker Desktop or [Colima](https://github.com/abiosoft/colima)
+1. **Clone the repository**: Clone this repository and navigate to the platformatic directory.
+
+```sh
+git clone https://github.com/platformatic/platformatic.git
+cd platformatic
+```
+
+2. **Install pnpm**: Install pnpm globally.
```

Comment:
Can we refer to the pnpm instructions instead?

Reply:
this is done.